### ISSUE TITLE
Add LiveQuery as module

### ISSUE THREAD

**Issue by mtrezza** [parse-community/Parse-SDK-iOS-OSX#1714]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-iOS-OSX/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-iOS-OSX/issues?q=is%3Aissue).

### Current Limitation

The LiveQuery SDK is separate from the main SDK. Since the main SDK has SPM support, LiveQuery can be added as a module.

### Feature / Enhancement Description

Add LiveQuery as a module.

**Comment by project-academy** [comment]
Did some light testing, looks like things are working - thank you!

**Comment by mtrezza** [comment]
Amazing, thanks for the feedback, @project-academy; if you could do some more testing and/or find anything weird, please let us know and open an issue.

**Comment by mtrezza** [comment]
Amazing, thanks for the feedback, @project-academy; if you could do some more testing and/or find anything weird, please let us know and open an issue.

**Comment by mtrezza** [comment]
Amazing, thanks for the feedback, @project-academy; if you could do some more testing and/or find anything weird, please let us know and open an issue. We'll go ahead an archive the other repository.

cc @dplewis

**Comment by mtrezza** [comment]
@sidan5, @extnous, @bayareahank, @owais-applligent, @ChrisCoder9000, @ULSashido, @ritteraf, @HackShitUp, @432player it would be great if you could also test this LiveQuery module out and let us know any feedback

**Comment by joaquin102** [comment]
Hey everyone.

I installed this module from 2.7.0 and right out of the box there is an issue with `WebSocketDelegate`

I am attaching here the screenshot but they main 2 issues are:
- It seems the delegate changed on the startscream library.
- Also they added an extra webSocketEvent called -> peerClosed

<img width="1146" alt="Screenshot 2023-09-04 at 1 00 28 AM" src="https://github.com/parse-community/Parse-SDK-iOS-OSX/assets/8617373/b61f83f5-c324-4963-a9b4-08da8a56ef0a">

**Comment by mtrezza** [comment]
@vazarkevych Could you take a look at this?

cc @dplewis Wasn't this an issue you've mentioned recently in a chat?

**Comment by project-saf** [comment]
I noticed this too - it's a change that was made in Starscream v4.0.6
The workaround in the meantime is to add Starscream as a package to your project, and set to the exact version v4.0.4
https://github.com/daltoniam/Starscream.git

**Comment by dplewis** [comment]
@mtrezza I moved the project but never used in production unfortunately.

@cbaker6 I’m still learning swift can you help out?

**Comment by joaquin102** [comment]
Sure @mtrezza I'll give it a try!

I can also confirm that the workaround suggested by @project-saf worked👍. The error is gone.

Also, and my apologies if this is not the right channel to post this question (I can delete it if this is not appropriate here), but I'm trying to implement ParseLiveQuery on my iOS app but I can't make it to work. I've spent days on this but I always get the error **cannot decode json into** 
I feel this is a silly error on my end. Does anyone know about this? I posted a full description on **stackoverflow**:

https://stackoverflow.com/questions/77046879/unable-to-get-parse-ios-sdk-livequery-to-work-always-get-cannot-decode-json-in 


Thank you for your help!
